src/code-components/FileInput/FileInput.register.ts

```diff
@@ -0,0 +1,65 @@
+import type { PlasmicLoader } from "../../plasmic";
+import { FileInput } from "./FileInput";
+
+export const FILE_TYPES = {
+  any: [],
+  images: ["image/*"],
+  videos: ["video/*"],
+  documents: [
+    ".pdf",
+    ".doc",
+    ".docx",
+    ".xml",
+    ".xls",
+    ".xlsx",
+    ".ppt",
+    ".pptx",
+    ".txt",
+    ".odt",
+    "application/msword",
+  ],
+};
+
+export function registerFileInput(
+  plasmic: PlasmicLoader,
+  modulePath = "@fullstackhouse/plasmic-utils/dist",
+) {
+  plasmic.registerComponent(FileInput, {
+    name: "FileInput",
+    importPath: modulePath + "/code-components/FileInput/FileInput",
+    props: {
+      onChange: {
+        type: "eventHandler",
+        advanced: true,
+        argTypes: [
+          {
+            name: "files",
+            type: "object",
+          },
+        ],
+      },
+      types: {
+        displayName: "Acceptable File Types",
+        type: "choice",
+        multiSelect: true,
+        options: Object.keys(FILE_TYPES),
+        defaultValue: ["any"],
```

Comment:
can we use `undefined` or `null` for an empty value? Rather than arbitrary string value such as `"any"`.